### ISSUE TITLE
Error during genome retrieval

### ISSUE THREAD

**Issue by Ste40cm** [ropensci/biomartr#77]
Hi, 
I have this issue for some weeks. I can't retrieve genomes or GFFs from FTP repository.
How did you solve this?

Here is an example of a failed genome retrieval of Escherichia_coli_O157:H7_str._Sakai

```
The FTP link: 'https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/008/865/GCF_000008865.2_ASM886v2/GCF_000008865.2_ASM886v2_genomic.fna.gz' seems not to be available at the moment. This might be due to an instable internet connection, a firewall issue, or wrong organism name. Could you please try to re-run the function to see whether it works now?
The FTP link: 'https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/008/865/GCF_000008865.2_ASM886v2/md5checksums.txt' seems not to be available at the moment. This might be due to an instable internet connection, a firewall issue, or wrong organism name. Could you please try to re-run the function to see whether it works now?
Genome download of Escherichia_coli_O157:H7_str._Sakai is completed!
The download session seems to have timed out at the FTP site 'https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/008/865/GCF_000008865.2_ASM886v2/GCF_000008865.2_ASM886v2_genomic.fna.gz'. This could be due to an overload of queries to the databases. Please restart this function to continue the data retrieval process or wait for a while before restarting this function in case your IP address was logged due to an query overload on the server side.
Error in readr::write_tsv(doc, file = file.path(path, paste0("doc_", local.org,  : 
  unused argument (file = file.path(path, paste0("doc_", local.org, "_db_", db, ".tsv")))
```

**Comment by HajkD** [comment]
Hi,

Did you use the recent developer version? I fixed a few issues in the past few days (see [NEWS](https://github.com/ropensci/biomartr/blob/master/NEWS.md)).

For example, using the new developer version of `biomartr` I could download all files using:

```r
biomartr::getCollection(organism = "GCF_000008865.2", db = "refseq")
```

If the issue still remains, could you please construct an example, so that I can reproduce your error?

I hope this helps!

Cheers,
Hajk

**Comment by Ste40cm** [comment]
Hi Hajk,
Thanks for the quick response. I'm using the latest version (1.0.2), Here's my example:

`getGenome( db="refseq", organism="Escherichia coli O157:H7 str. Sakai" )
`

here is the error I get:
```
Starting genome retrieval of 'Escherichia coli O157:H7 str. Sakai' from refseq ...


|==========================================================================================================================| 100% 68 MB
The FTP link: https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/008/865/GCF_000008865.2_ASM886v2/GCF_000008865.2_ASM886v2_genomic.fna.gz' seems to be unavailable at the moment. This could be due to an unstable internet connection, a firewall problem or the wrong body name. Could you please try re-running the function to see if it works now?
The FTP link: https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/008/865/GCF_000008865.2_ASM886v2/md5checksums.txt' seems to be unavailable at the moment. This could be due to an unstable internet connection, a firewall issue, or the wrong organism name. Could you try re-running the function to see if it works now?
The download of the Escherichia_coli_O157:H7_str._Sakai genome is complete!
The download session seems to have expired at the FTP site 'https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/008/865/GCF_000008865.2_ASM886v2/GCF_000008865.2_ASM886v2_genomic.fna.gz'. This may be due to database query overload. Please restart this function to continue the data retrieval process or wait a while before restarting this function in case your IP address was logged due to server-side query overload.
Error in readr::write_tsv(doc, file = file.path(path, paste0("doc_", local.org, : 
  unused argument (file = file.path(path, paste0("doc_", local.org, "_db_", db, ".tsv"))

```
Thanks.

**Comment by HajkD** [comment]
So you installed the [developer version](https://github.com/ropensci/biomartr#install-developer-version) within the past 3 days (where I fixed existing errors, see #76) and tried running:

```r
biomartr::getGenome(organism = "GCF_000008865.2", db = "refseq")
```

(the organism name `Escherichia coli O157:H7 str. Sakai` doesn't follow the scientific naming convention as will not be recognized)

In addition, you could set:

```r
options(timeout = 30000000)
```

If all this doesn't work, then there should be a firewall or internet access issue via R on you side.

Cheers,
Hajk
